workshops/Tkinter Workshop/README.md

```diff
@@ -0,0 +1,249 @@
+
+```python
+def clicked(quote):
+```
+
+In this function, we are showing a new page in which a random quote is displayed. We are using the Frame function to create a page here.
+
+```python
+F1=tk.Frame(window,bg="light salmon",width=800,height=560)
+F1.place(x=0,y=0)
+```
+
+The Frame function is used to organize a group of widgets. It acts like a container that can be used to hold the other widgets. The rectangular areas of the screen are used to organize the widgets in the python application. And in that frame, we display the quote at the center using the label function.
+
+```python
+tk.Label(F1,text=quote,font=("Helvetica", 30,"bold"),bg="light salmon",fg="black").pack()
+```
+
+We have to pass the random quote to this function.
+
+```python
+text=quote
+```
+
+Here we gave text as the random quote passed from the button.
+
+If you don't understand, just go to the live demo and see the button workflow [here](https://repl.it/@bezlin/graphics#main.py).
+
+#### Finally the code will look like this.
+
+```python
+import tkinter as tk
+import random
+
+quotelist = [
+
+ "Instead of wondering when your \nnext vacation is, maybe you \nshould set up a life you don’t \nneed to escape from.",
+
+ "Sometimes\n you win,\n sometimes \nyou learn.",
+
+ "Avoiding failure\n is to avoid \nprogress.",
+
+ "If the plan \ndoesn’t work, \nchange the plan,\n but never the \ngoal.",
+
+ "Creativity \nis \nintelligence \nhaving \nFun."
+
+ ]
+
+def clicked(quote):
+ F1=tk.Frame(window,bg="light salmon",width=800,height=560)
+ F1.place(x=0,y=0)
+ tk.Label(F1,text=quote,font=("Helvetica", 30, "bold"),bg="light salmon",fg="black").pack()
+window = tk.Tk()
+window.minsize(800, 560)
+window.title("Quotes Hub")
+window.config(bg="black")
+tk.Label(window, font=("Helvetica", 60, "bold"),
+ text="QUOTES HUB", bg="black", fg="white").pack()
+tk.Button(window,command=lambda:clicked(random.choice(quotelist)),text="Open",bg="aqua",fg="black",width=20).place(x=315,y=300)
+window.mainloop()
+
+```
+
+It’s simple, you just have to go through the functions and workflow.
+
+![ezgif com-gif-to-mp4](https://cloud-5m2nwfs8r.vercel.app/ezgif.com-video-to-gif-4.gif)
+
+## Hacking time!
+
+Now you know how to use Tkinter to make UI. You should not stop here. You need to learn more You can refer to the documentation [Learn Tkinter](https://docs.python.org/3/library/tk.html). The best way to learn it is to make real-world projects
+
+Lastly, don't get stuck.
+
+I am attaching the Tkinter documentation take a look at [Learn Tkinter](https://docs.python.org/3/library/tk.html).
+Also if you are new to python, take a look at [python](https://www.python.org/doc/) documentation.
+
+### Applications made using Tkinter by other hackers.
+
+[Hariprasad Rajan](https://repl.it/@HariprasadR03/haroadas#main.py)
```

Comment:
I've mentioned this a couple of times but would it be possible for these to describe whats unique about each demo so the reader knows what to look for? It's only got to be about half a sentence

Reply:
In these the other hackers have build their veiws on top of it

Reply:
gotcha, could you add that to the workshop?

Reply:
It shows people who are viewing this workshop on how expandable it can be. All hackers have integrated some features like the back button, exit button, Never end quotes, changed quote hub to a movie summary slideshow, Quotes of famous people by selecting them. All these shows how expandable the workshop is.

Reply:
ik, could you add that to the workshop?

Reply:
This isn't what I meant. I meant that this could be a way for you to provide context to each demo as an example.

Reply:
ok